Source/SIMPLib/DataArrays/DataArray.cpp

```diff
@@ -1447,17 +1457,8 @@ void DataArray<T>::deallocate()
       }
     }
   }
-#if 0
-      if (MUD_FLAP_0 != 0xABABABABABABABABul
```

Comment:
Let's think twice about getting rid of this little nugget of code. 99.9% of the time we don't need it, but when we do it is nice to just change the #if to 1 and recompile.